site/src/docs/components/select/index.mdx

```diff
@@ -0,0 +1,250 @@
+  <Select
+    multiSelect
+    filter={defaultFilter}
+    texts={{
+      label: 'Select multiple options',
+      placeholder: 'Choose many',
+    }}
+    options={[{ label: 'Americium' }, { label: 'Berkelium' }, { label: 'Californium' }, { label: 'Copernicium' }]}
+    onChange={(selectedOptions) => {
+      console.log('selectedOptions:' + selectedOptions.map((option) => option.value).join(', '));
+    }}
+  />
+</PlaygroundPreview>
+
+#### Multiselect with filter and groups
+
+Options can be grouped to provide additional context or to visually separate them. Each group requires a title for better organization. A group can also allow the user to select all items by naming it “Select All” with a checkbox.
+
+<PlaygroundPreview>
+  <Select
+    multiSelect
+    filter={defaultFilter}
+    texts={{
+      label: 'Select multiple options',
+      placeholder: 'Choose many',
+    }}
+    groups={[
+      {
+        label: 'Group 1 (A-C)',
+        options: [{ label: 'Americium' }, { label: 'Berkelium' }, { label: 'Californium' }, { label: 'Copernicium' }],
+      },
+      {
+        label: 'Group 1 (E-P)',
+        options: [{ label: 'Einsteinium' }, { label: 'Fermium' }, { label: 'Mendelevium' }, { label: 'Plutonium' }],
+      },
+    ]}
+    onChange={(selectedOptions) => {
+      console.log('selectedOptions:' + selectedOptions.map((option) => option.value).join(', '));
+    }}
+  />
+</PlaygroundPreview>
+
+#### Multiselect with preselections
+
+When options are selected in multiselect, they appear below the select field. The user can clear selected options individually or all at once using a “Clear All” button.
+
+<PlaygroundPreview>
+  <Select
+    multiSelect
+    filter={defaultFilter}
+    texts={{
+      label: 'Select multiple options',
+      placeholder: 'Choose many',
+    }}
+    groups={[
+      {
+        label: 'Group 1 (A-C)',
+        options: [
+          { label: 'Americium', selected: true },
+          { label: 'Berkelium' },
+          { label: 'Californium' },
+          { label: 'Copernicium' },
+        ],
+      },
+      {
+        label: 'Group 1 (E-P)',
+        options: [
+          { label: 'Einsteinium' },
+          { label: 'Fermium', selected: true },
+          { label: 'Mendelevium', selected: true },
+          { label: 'Plutonium' },
+        ],
+      },
+    ]}
+    onChange={(selectedOptions) => {
+      console.log('selectedOptions:' + selectedOptions.map((option) => option.value).join(', '));
+    }}
+  />
+</PlaygroundPreview>
+
+#### Multiselect with tags somewhere else
+
+Selected options can be displayed in another section of the page. This approach allows options to be grouped in the same location with multiple select components, such as in filtering search pages.
+
+An example of this complex variant is <ExternalLink href="https://hds.hel.fi/storybook/react/?path=/story/components-select--multiselect-with-custom-tags">in the Storybook.</ExternalLink>
+
+### Keyboard usage
```

Comment:
Should we move keyboard usage section under accessibility tab? I see usage tab more for HDS user usage guidelines and not so much for end user usage.

Reply:
I had this first in the "code.mdx", but confluence had in front page, so moved it there.

Users with screen readers have mostly different keyboard commands for navigation (shift+alt+ctrl + arrow down, etc), so these are more for non-screen reader users, so maybe not so much "accessibility" tab stuff?

Reply:
Let's keep it as it is now and figure out common place for keyboard usage later if needed.